public/static/docs/command-reference/remote/add.md

```diff
@@ -226,6 +226,24 @@ On successful access token generation, token data will be cached in git ignored
 directory with path `.dvc/tmp/gdrive-user-credentials.json`. Do not share token
 data with anyone else to prevent unauthorized access to your Google Drive.
 
+#### Shared drives support
```

Comment:
Not sure about adding subheaders in hidden sections. We don't even have H4s anywyere yet, this would be the first one in all the docs actually (except for 2 wrong ones I just noticed in `dvc version` which I'll fix soon). The anchor icon is not displayed properly:
![image](https://user-images.githubusercontent.com/1477535/72213324-982b0280-34b2-11ea-8989-22cd5a65112d.png)
and if you try to use the direct link (https://dvc.org/doc/command-reference/remote/add#shared-drives-support) the page won't open the section and scroll to that header.

Reply:
Hm I just found another one in `dvc remote add` actually, inside the **Aliyun OSS** section.

Anyway, since this is merged I'll just come up with a better solution in a near-future "regular updates" PR.

Reply:
@jorgeorpinel yep, we can change it to a regular **bold**

Reply:
OK addressed. Feel free to review: 26d9928b Thanks